packages/@okta/vuepress-site/docs/concepts/oauth-claims/index.md

```diff
@@ -0,0 +1,125 @@
+
+The `scp` (scope) parameter passes two scopes to the resource server: `openid` and `email`. The `email` scope contains the `email` and `email_verified` claims and these claims provide access to the user's email address and its verification status. The `openid` scope indicates that the token is part of an OIDC request.
+
+See [Access token scopes and claims](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#access-token-scopes-and-claims) for information about configuring scopes and claims in access tokens.
+
+### Claims in ID tokens
+
+Claims in ID tokens pass identifying information about a user to client apps. The identifying information is structured into standard claims that the [OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) defines. Client apps evaluate the claims and make authorization decisions based on that information.
+
+The claims that you pass in an ID token depend on your use case. In an authentication context, you can pass claims that provide more details about a user, such as their `family_name`, `birthdate`, or `address`. You can use these claims to personalize the user experience in a client app. Or you can use them for enhanced [identity verification](/docs/guides/idv-integration/main/#supported-oidc-claims).
+
+See the following example of an ID token payload.
+
+```json
+{
+  "sub": "user123",
+  "email": "user@example.com",
+  "phone_number": "+1234567890",
+  "name": "John Doe"
+}
+```
+
+In this example, the ID token payload passes the user ID (`sub`), user's email (`email`), phone number (`phone_number`), and their name (`name`). These claims identify the user to the client app.
+
+See [ID token](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/overview/#id-token) for information about how ID tokens are structured and used in Okta.
+
+## Claim types
+
+Claims can be categorized into two groups based on how they're defined and used.
+
+**Registered claims**: There are seven registered claims that are defined in the [JWT spec](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1). They provide fundamental information like the token's issuer (`iss`), the user's unique ID (`sub`), or when the token expires (`exp`).
+
+**Custom claims**: Custom claims consist of [public claims](#public-claims) that third-parties have registered and [private claims](#private-claims) that are created for use between specific parties.
+
+### Public claims
+
+Public claims are designed to avoid naming collision with other registered claims. Public claims include the [OIDC standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). For a full list of public claims, see the [Internet Assigned Numbers Authority (IANA) registry](https://www.iana.org/assignments/jwt/jwt.xhtml).
```

Comment:
```suggestion
Public claims are designed to avoid naming collisions with other registered claims. Public claims include the [OIDC standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). For a full list of public claims, see the [Internet Assigned Numbers Authority (IANA) registry](https://www.iana.org/assignments/jwt/jwt.xhtml).
```